### PR TITLE
update styles and refactor out repeated html

### PR DESCRIPTION
**Pull request by noobjey**
- partial for page titles so they all have the same formatting
- added styles to login form
- updated session controller to pass in a different flash message type to conform to bootsrap alerts
- updated user controller to pass in a different flash message type to conform to bootstrap alerts
- added styles to sign up form
- sign up form stays on page when its unsuccessful and feature test
- added styles to category page
- updated all feature tests for html css

closes #45

### CODE REVIEW

**Comment by applegrain** [comment]
Awesome! @mrjaimisra do you want to merge?

**Comment by noobjey** [comment]
Hey guys, I created lots of hound violations for the format of the feature test i wrote: user_signup_spec.rb. I like white space, let me know if you guys think its to much and i will correct it.

File: app/views/layouts/_flash_message.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <% flash.each do |type, message| %>
-    <div class="alert alert-success" role="alert">
+    <div class="alert alert-<%= type %>" role="alert">
```
Comment (applegrain) [inline app/views/layouts/_flash_message.html.erb]: Should it be in erb tags or just in string interpolated?

File: spec/features/user_signup_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
```
Comment (applegrain) [inline spec/features/user_signup_spec.rb]: Nice feature test!

File: spec/features/user_views_items_by_category_spec.rb [hunk 3]
```diff
@@ -2,13 +2,27 @@
 
 RSpec.describe "the category view", type: :feature do
   context "user visits a category page" do
-    let!(:item) { Item.create(name: "Breakfast burrito",
-                              description: "delicious",
-                              price: 5, image_url: test_image_url,
-                              category_id: 1) }
-
     let!(:category) { Category.create(name: "breakfast") }
+    let!(:item) {
```
Comment (applegrain) [inline spec/features/user_views_items_by_category_spec.rb]: I like the whitespace.. hound just has to deal with it.

File: app/views/layouts/_flash_message.html.erb (hunk 1 above)
Comment (noobjey) [inline app/views/layouts/_flash_message.html.erb]: Im not sure i understand the question. Can you show me an example? Or just hit me up on slack.

Comment (applegrain) [inline app/views/layouts/_flash_message.html.erb]: My bad! I was thinking of an instance where the entire thing was an erb tag.

**Pull request merged**